### ISSUE TITLE
How can use this, or any step I missed ? 

### ISSUE THREAD

**Issue by OhIAmFine** [bigH/git-fuzzy#6]
I have installed fuzy, and followed the installtion step, but i can't run.
![image](https://user-images.githubusercontent.com/10390004/83402286-060fa080-a439-11ea-8b07-1bafc2166ceb.png)

**Comment by baklazaner** [comment]
I dont see your config file name, but lets say it is `.zshrc`. Then type this or restart terminal.

`source ~/.zshrc`

**Comment by jonleopard** [comment]
I tried installing `git-fuzzy`  as a zsh plugin (with [antibody](https://getantibody.github.io/usage/)) and it doesn't seem to be sourcing for me either. I've confirmed that the binary is in my `PATH`, but when I type `get fuzzy`, the following error returns:
`git: 'fuzzy' is not a git command. See 'git --help'.`

Has anyone else here successfully installed this as a zsh plugin with their plugin manager?
